security-privacy-questionnaire.md

```diff
@@ -0,0 +1,111 @@
+# Security and Privacy Questionnaire
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+WebXR Hit Test Module specification.
+
+**What information might this feature expose to Web sites or other parties,
+and for what purposes is that exposure necessary?**
+
+This feature will help a web site to map the user environment. Hit tests, in the
+context of VR will only help map the 3D environment. However, in the context of
```

Comment:
I think that depending on the exact implementation, the VR headsets could also potentially map the real world around the user. The initial implementations would probably leverage the headsets' information of the 3D play area, but it seems to me that spec-compliant implementation on VR could do the same thing that AR implementation is doing.

Reply:
I guess VR headsets could expose play areas and stuff like that and expose it with hit tests. Would you like me to mention this? (Running to a meeting so I can do that afterwards.)

Reply:
Applied comment by dropping AR/VR distinction.